lib/Textmaster/Api/Locale.php

```diff
@@ -62,4 +65,34 @@ public function countries($locale)
     {
         return $this->get('public/countries/'.rawurlencode($locale));
     }
+
+    /**
+     * List all available locales for an ability.
+     *
+     * @param string      $ability Can be one of ProjectInterface::ACTIVITY
+     * @param int         $page
+     * @param string|null $locale
+     *
+     * @return array
+     */
+    public function abilities($ability, $page, $locale = null)
+    {
+        $abilities = ProjectModel::getAllowedActivities();
```

Comment:
You shouldn't rely on the model from within the Api.... I think these constants should be moved to the Api layer or to a standalone class. Maybe using https://github.com/myclabs/php-enum.
what do you think?